.gitignore

```diff
@@ -1,3 +1,4 @@
 ca.crt
 ca.key
 *.t.err
+*~
```

Comment:
That's because of `sed -i~` (only way to make sed compatible on both macOS and Linux). But it seems like I do not use `sed -i~` anywhere, so this line can be removed.